acima-credit/denv
# Add `from_hash` To List Of Available Sources

## Ticket
https://acima.atlassian.net/browse/ATL-10480

## Summary
I'm adding a `from_hash` method to the available sources.

## Details
This will allow the consumer to load env settings from a hash, not just from a file.

## How to test
1. Update Merchant Portal to point to this version.
```ruby
gem "denv",   path: "path-to-local-denv-branch"
```
2. Start a Merchant Portal rails console.
```ruby
hash = { one: 1, two: 2, three: 3 }
DEnv.from_hash!(hash)
puts ENV["one"]
```
3. Expected Result
This will output `1`.

This will also be tested in MP via 
https://github.com/acima-credit/merchant_portal/pull/9424
